utils/travis/publish-gh-pages.sh

```diff
@@ -2,5 +2,5 @@
 set -x
 set -e
 
-yarn build:demo --since HEAD^
+yarn build:demo
```

Comment:
With the `--since` changes in Lerna, this is no longer building the most recent CHANGELOG updates for each package. This will just build everything like last time.